scripts/helpers/plugin_functions.sh

```diff
@@ -70,7 +70,10 @@ tpm_path() {
 
 tpm_plugins_list_helper() {
 	# lists plugins from @tpm_plugins option
-	echo "$(tmux start-server\; show-option -gqv "$tpm_plugins_variable_name")"
+	local plugins
+	plugins="$(tmux start-server\; show-option -gqv "$tpm_plugins_variable_name")"
```

Comment:
Can you please make sure the comment `# lists plugins from @tpm_plugins option` is above this line?